packages/@aws-cdk/aws-logs/lib/data-protection-policy.ts

```diff
@@ -0,0 +1,180 @@
+        bucket: props.s3AuditDestination,
+      };
+    }
+
+    if (props.firehoseAuditDestination) {
+      findingsDestination.firehose = {
+        deliveryStream: props.firehoseAuditDestination,
+      };
+    }
+
+    var identifierArns: string[] = [];
+    for (let identifier of props.identifiers) {
+      identifierArns.push('arn:aws:dataprotection::aws:data-identifier/' + identifier);
+    };
+
+    var auditStatement: AuditStatement = {
+      sid: 'audit-statement-cdk',
+      dataIdentifier: identifierArns,
+      operation: {
+        audit: {
+          findingsDestination: findingsDestination,
+        },
+      },
+    };
+
+    var redactStatement: RedactStatement = {
+      sid: 'redact-statement-cdk',
+      dataIdentifier: identifierArns,
+      operation: {
+        deidentify: {
+          maskConfig: {},
+        },
+      },
+    };
+
+    this.statement = [auditStatement, redactStatement];
+  }
+
+}
+
+type AuditStatement = {
+  sid: string;
+  dataIdentifier: string[];
+  operation: AuditOperation;
+}
+
+type AuditOperation = {
+  audit: FindingsDestinations;
+}
+
+type FindingsDestinations = {
+  findingsDestination: FindingsDestination;
+}
+
+type FindingsDestination = {
+  cloudWatchLogs?: CloudWatchLogsDestination;
+  firehose?: FirehoseDestination;
+  s3?: S3Destination;
+}
+
+type CloudWatchLogsDestination = {
+  logGroup: string;
+}
+
+type FirehoseDestination = {
+  deliveryStream: string;
+}
+
+type S3Destination = {
+  bucket: string;
+}
+
+type RedactStatement = {
+  sid: string;
+  dataIdentifier: string[];
+  operation: DeidentifyOperation;
+}
+
+type DeidentifyOperation = {
+  deidentify: MaskConfig;
+}
+
+type MaskConfig = {
+  maskConfig: {}
+}
+
+
+/**
+ * Interface for creating a data protection policy
+ */
+export interface DataProtectionPolicyProps {
+  /**
+   * Name of the data protection policy
+   *
+   * @default - 'data-protection-policy-cdk'
+   */
+  readonly name?: string;
+
+  /**
+   * Description of the data protection policy
+   *
+   * @default - 'cdk generated data protection policy'
+   */
+  readonly description?: string;
+
+  /**
+   * List of data protection identifiers. Must be in the following list: https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/protect-sensitive-log-data-types.html
+   */
+  readonly identifiers: string[];
+
+  /**
+   * CloudWatch Logs log group to send audit findings to. The log group must already exist.
+   *
+   * @default - no CloudWatch Logs audit destination
+   */
+  readonly cloudWatchLogsAuditDestination?: string;
+
+  /**
+   * S3 bucket to send audit findings to. The bucket must already exist.
+   *
+   * @default - no S3 bucket audit destination
+   */
+  readonly s3AuditDestination?: string;
+
+  /**
+   * Amazon Kinesis Data Firehose stream to send audit findings to. The delivery stream must already exist.
+   *
+   * @default - no firehose delivery stream audit destination
+   */
+  readonly firehoseAuditDestination?: string;
```

Comment:
As above, this should be a IDeliveryStream.

Reply:
See comment above.